packages/tracing-internal/tsconfig.types.json

```diff
@@ -4,11 +4,7 @@
   // the fact that it introduces a dependency on `@sentry/browser` which doesn't exist anywhere else in the SDK, which
   // then prevents us from building that and this at the same time when doing a parallellized build from the repo root
   // level.
-  "exclude": [
-    "src/index.bundle.ts",
-    "src/index.bundle.feedback.ts",
-    "src/index.bundle.replay.ts"
-  ],
+  "exclude": ["src/index.bundle.ts", "src/index.bundle.feedback.ts", "src/index.bundle.replay.ts"],
```

Comment:
ok so biome is now putting constructor arguments in one line each but these array values now all get thrown into one line? Kinda worse than before IMO but I don't wanna spend too much time arguing over formatting so it's fine.

Reply:
I think Biome is just fixing Prettier compatibility right now.

Reply:
Also, we weren't parsing this file previously, because it contains comments, and now we do.

Reply:
oh I see, makes sense, thx